onnxruntime/core/providers/cpu/tensor/upsample.cc

```diff
@@ -509,7 +517,19 @@ void ResizeBiCubic(
         }
 
         auto y_int = static_cast<int64_t>(std::floor(in_y));
-        auto& coeff_y = cubic_coeffs[in_y - y_int];
+        auto& coeff_y = exclude_outside ? y_coeff_holder : cubic_coeffs[in_y - y_int];
+        y_coeff_sum = 1;
+
+        if (exclude_outside) {
+          // When true, the weight of sampling locations outside the grid will be set to 0
```

Comment:
>0 [](start = 89, length = 1)

nit: 0.f

Reply:
sure... I will defer nits to a separate PR given this PR is targeted for 1.0